conjur/logic/logout_logic.py

```diff
@@ -6,19 +6,22 @@
 This module is the business logic for logging out of the Conjur CLI
 """
 # pylint: disable=too-few-public-methods
+from conjur.interface.credentials_store_interface import CredentialsStoreInterface
```

Comment:
this should be above the previous line. `# pylint: disable=too-few-public-methods` should be above the `class LogoutLogic`

Reply:
done